.github/workflows/release-js.yaml

```
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+
+name: Release JS Library
+run-name: Publish Release ${{github.ref_name}} JS Library
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: false
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # ex. v1.0.0
+      - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+" # ex. v1.1.0-rc1
+      - "v0.0.1" # used for testing only
+      - "v0.0.1-rc[0-9]+" # used for testing only
+
+env:
+  NEW_RELEASE_TAG: ${{github.ref_name}}
+  TEST_RUN: ${{startsWith(github.ref_name, 'v0.0.1')}}
+
+jobs:
+  publish-npm:
+    name: Publish NPM Package
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [20.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache-dependency-path: package-lock.json
+
+      - name: Install dependencies
+        working-directory: .
+        run: npm i
+
+      - name: Version Package
+        if: env.TEST_RUN != 'true'
+        working-directory: .
+        run: npm version --new-version "${{env.NEW_RELEASE_TAG}}" --no-git-tag-version
+
+        # Required for Granular Access Tokens
+      - name: NPM Access Config
+        run: npm config set //registry.npmjs.org/:_authToken ${{secrets.NPM_AUTH_TOKEN}}
```

Different named variable for this org:
```suggestion
        run: npm config set //registry.npmjs.org/:_authToken ${{secrets.NODE_AUTH_TOKEN}}
```